### PR TITLE
[1.0.0] Add sample commands for ELK installation

### PR DESCRIPTION
**Pull request by sgayangi**

### CODE REVIEW

**Review by CrowleyRajapakse — APPROVED** [review]

File: en/docs/setup/analytics/configure-analytics-for-elk-stack.md [hunk 1]
```diff
@@ -2,29 +2,51 @@
 
 ## Step 1 - Setup APK
 
-1. Start by following the instructions outlined in [Customize Configurations](../Customize-Configurations.md). These instructions will guide you through the process of acquiring the `values.yaml` file, which you will then use to tailor the analytics configurations to your specific needs. Open `values.yaml` file.
+1. Start by following the instructions outlined in [Customize Configurations](../Customize-Configurations.md). These instructions will guide you through the process of acquiring the `values.yaml` file.
+2. Open the `values.yaml` file, and add the above configuration to the gatewayRuntime section under dp.
```
Comment (YasasRangika) [inline en/docs/setup/analytics/configure-analytics-for-elk-stack.md:6]: add the ~above~ below configuration

File: en/docs/setup/analytics/configure-analytics-for-elk-stack.md [hunk 2]
```diff
@@ -2,29 +2,51 @@
...
+        enabled: true
+        publishers:
+        - enabled: true
+          type: "elk"
+          logLevel: "INFO"
     ```
 
-    !!! Note
-        Optionally, `logLevel`. By default, this config is set to `INFO`.
+Your values.yaml file will have a structure as follows.
```
Comment (YasasRangika) [inline en/docs/setup/analytics/configure-analytics-for-elk-stack.md:30]: Now the structure of your values.yaml file will be as follows:

Comment (sgayangi) [reply en/docs/setup/analytics/configure-analytics-for-elk-stack.md:30]: Fixed in https://github.com/wso2/docs-apk/pull/584

File: en/docs/setup/analytics/configure-analytics-for-elk-stack.md (hunk 1 above)
Comment (sgayangi) [reply en/docs/setup/analytics/configure-analytics-for-elk-stack.md:6]: Fixed in https://github.com/wso2/docs-apk/pull/584

Sentence changed to: Open the `values.yaml` file and set the following configuration under the `wso2.apk.dp.gatewayRuntime` section.

**Pull request merged**